### ISSUE TITLE
Default value for files option does not work

### ISSUE THREAD

**Issue by saschask** [webpack-contrib/stylelint-webpack-plugin#169]
* Operating System: Windows 10 Enterprise
* Node Version: 10.14.0
* NPM Version: 6.5.0
* webpack Version: 4.26.1
* stylelint-webpack-plugin Version: 0.10.5

This issue is for a:

- [x] **bug**
- [ ] **feature** request
- [ ] **modification** request

##### CLI Command

```bash
  $ webpack
```

##### webpack.config.js

```js
const stylelintWebpackPlugin = require('stylelint-webpack-plugin');
const stylelintFormatterPretty = require('stylelint-formatter-pretty');
// ...

module.exports = {
	// ...
	plugins: [
		// ...
		new stylelintWebpackPlugin({
			context: path.resolve(__dirname, 'source'),
			formatter: stylelintFormatterPretty
		}),
		// ...
	],
	// ...
};
```

### Expected Behavior

Since the default value for the `files` option is `'**/*.s?(a|c)ss'`, all my `*.css` files inside the `source` folder should automatically be found by the plugin. `__dirname` represents the location of the `webpack.config.js`. `source` is located inside `__dirname` (one folder level below).

### Actual Behavior

None of my `*.css` files can be found by the plugin. I need to change the `files` option of the plugin to a value with a specific file extension in order to get it working as expected:

```js
const stylelintWebpackPlugin = require('stylelint-webpack-plugin');
const stylelintFormatterPretty = require('stylelint-formatter-pretty');
// ...

module.exports = {
	// ...
	plugins: [
		// ...
		new stylelintWebpackPlugin({
			context: path.resolve(__dirname, 'source'),
			files: '**/*.css', // with this line it works as expected
			formatter: stylelintFormatterPretty
		}),
		// ...
	],
	// ...
};
```

Is it possible that the default `files` option `'**/*.s?(a|c)ss'` is not a valid glob pattern and can therefore not be used to find the files?

### How Do We Reproduce?

1. Add and configure the plugin as described above in the **Code** section.
2. Put some `*.css` files with stylelint errors at the respective location in your project.
3. Check whether stylelint results are shown in the console or not.

Thanks for having a look at it and giving some feedback.

Best Sascha

**Comment by leevigraham** [comment]
In my case `files: ["assets/**/*.scss"]` works `files: ["assets/**/*.s?css"]` fails.

**Comment by leevigraham** [comment]
@saschask Which version of stylelint are you using?

**Comment by saschask** [comment]
@leevigraham When I created this issue I was using`"stylelint": "9.9.0"` in parallel. Have not double-checked with newer versions so far.

**Comment by leevigraham** [comment]
@saschask There's a few other issues that say this plugin is not compat with v9… that could be one issue. The glob pattern that worked for me was: `**/*.{css,scss}`

**Comment by ricardogobbosouza** [comment]
The option `files` is passed directly to `stylelint`.
Probably a `stylelint` issue depending on your version
